src/components/CreateMatch/Team/CreateTeam/CreateTeam.js

```diff
@@ -1,38 +1,49 @@
-import React from 'react';
-import addTeamList from '../../../../images/create-team-list.png';
-import deleteIcon from '../../../../images/delete-icon.png';
-import editIcon from '../../../../images/edit-icon.png';
-import newPlayer from '../../../../images/add-player.png';
-import './Team.scss';
+import React, { Component } from 'react';
 
-const CreateTeam = () => {
-    return (
-        <span>
-            <span>
-                <h1>Crea tu Equipo</h1>
-                <img scr={addTeamList} alt="adicionar lista jugadores"/>
-            </span>
-            <span>
-                <span>
-                    <span>
-                        <input placeholder="Nuevo Jugador"></input>
-                        <img scr={newPlayer} alt="adicionar jugador"/>
-                    </span>
-                    <span>
-                        <span>Tu Equipo</span>
-                        <span>
-                            <span>Andres</span>
-                            <img scr={editIcon} alt="editar jugador"/>
-                            <img scr={deleteIcon} alt="eliminar jugador"/>
-                        </span>
-                    </span>
-                    <span>
-                        <button>Guardar</button>
-                        <button>Cancelar</button>
-                    </span>
+import close from '../../../../images/delete-team.png';
+import ballPlay from '../../../../images/ball-play.png';
+import captain from '../../../../images/captain.png';
+import check from '../../../../images/check-icon.png';
+
+import './CreateTeam.scss';
+
+class CreateTeam extends Component {
+    constructor (){
+        super();
+        this.state = {
```

Comment:
if you are not going to use the state here please delete the constructor method.